Examples/Tests/load_external_field/analysis_time_scaling.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+# Compare B(component) scaling between two plotfiles:
+# checks that BN/B0 ~= expected_ratio on cells where |B0| > min_abs
+
+import argparse
+
+import numpy as np
+
+
+def load_plotfile_arrays(path, comp):
+    # Prefer AMReX's native plotfile reader; fall back to yt.
+    try:
+        from amrex import plotfile as amr
+
+        pf = amr.PlotFile(path)
+        arr = pf.get(comp, level=0)  # numpy array (nx, ny, nz) or (nr, nz)
+        t = float(pf.time())  # seconds
+        return arr, t
+    except ImportError:
+        import yt
+
+        ds = yt.load(path)
+        if hasattr(ds, "force_periodicity"):
+            ds.force_periodicity()
+        ad = ds.covering_grid(
+            level=0, left_edge=ds.domain_left_edge, dims=ds.domain_dimensions
+        )
+        arr = ad[("boxlib", comp)].squeeze().v
+        t = float(ds.current_time.in_units("s"))
+        return arr, t
```

Comment:
```suggestion
    import yt

    ds = yt.load(path)
    if hasattr(ds, "force_periodicity"):
        ds.force_periodicity()
    ad = ds.covering_grid(
        level=0, left_edge=ds.domain_left_edge, dims=ds.domain_dimensions
    )
    arr = ad[("boxlib", comp)].squeeze().v
    t = float(ds.current_time.in_units("s"))
    return arr, t
```

Reply:
None of our other tests are using the native amrex plotfile reader. For the sake of consistency, I would recommend to only use `yt` here.